pkgs/applications/misc/yubioath-flutter/default.nix

```diff
@@ -60,16 +56,16 @@ flutter.buildFlutterApplication rec {
     # Symlink binary.
     ln -sf "$out/app/authenticator" "$out/bin/yubioath-flutter"
 
-    # Needed for QR scanning to work.
```

Comment:
Might be helpful to keep this comment and add it before the `extraWrapProgramArgs`.